torch/csrc/jit/codegen/cuda/ir_internal_nodes.h

```diff
@@ -456,6 +460,41 @@ class TORCH_CUDA_CU_API IterDomain : public Val {
     return extent_;
   }
 
+  //! Dimension padding interface:
+  //!  2 modes are currently supported:
+  //!
+  //!   - mode 1: if to_size is given as a positive number,
+  //!      the dimension will be padded to the size so that
+  //!      this iterdomain will be compile-time constant
+  //!      size and it is the scheduler's responsibility
+  //!      to ensure no input larger than the padded size
+  //!      will be observed
+  //!
+  //!   - mode 2: if no to_size is given, this dimension
+  //!      is "dynamically" padded to the nearest multiple
```

Comment:
Is this nearest or the next largest multiple?

Reply:
I meant rounding up to the next multiple of 32, i.e. 25->32, 33->64. Sorry for the confusion. Will update,